assets/css/base/gutenberg-editor.scss

```diff
@@ -247,7 +247,6 @@ figure {
 img {
 	height: auto; // Make sure images are scaled correctly.
 	max-width: 100%; // Adhere to container width.
-	display: block;
```

Comment:
Isn’t this change a bit wildcard and might affect all image blocks? would it make sense to scope to cart only?